pkg/operatorrules/registry.go

```diff
@@ -68,7 +68,10 @@ func (r *Registry) ListAlerts() []promv1.Rule {
 	}
 
```

Comment:
you can use the standard library's slices.Clone function here; e.g 
```suggestion
alerts := slices.Clone(r.registeredAlerts)
```

Reply:
I think this would only work if registeredAlerts was a slice, it is a map to allow multiple calls to the register method without duplicating the alerts

Reply:
:+1: